scripts/specs.ts

```diff
@@ -100,6 +100,10 @@ const defaultAllowlist: allowlistItem[] = [
     [
         "https://github.com/tc39/proposal-import-attributes/tree/abca60286360b47f9a6be25a28f489c2cb157beb",
         "Allowed because import assertions were replaced in-place by import attributes. Remove this exception when javascript.statements.import.import_assertions is dropped from BCD in June 2026."
+    ],
+    [
+        "https://open-ui.org/components/customizableselect/",
+        "Allowed because customizable select doesn't yet have a spec URL. And even when it does, it will be defined in multiple HTML, CSS, and ARIA specs. The OpenUI explainer is the only central place that defines the component at the moment."
```

Comment:
Is there a good issue or PR to link to that would prompt us to remove this exception? Basically, I want to know where to look when we have "real" spec links to replace this with.

Something like this?

```suggestion
        "Allowed because customizable select doesn't yet have a spec URL. And even when it does, it will be defined in multiple HTML, CSS, and ARIA specs. The OpenUI explainer is the only central place that defines the component at the moment. Remove when SOMEURL merges."
```

Reply:
There are multiple PRs, not just one. See my previous comment: https://github.com/web-platform-dx/web-features/pull/2301#issuecomment-2624608170
Out of those, I'm guessing it would be enough to link to the following ones:

https://github.com/whatwg/html/pull/10586
https://github.com/whatwg/html/pull/10629
https://github.com/whatwg/html/pull/10633

@josepharhar are you able to help select one (or more, but ideally a small number) spec PR that would basically signify "that's it, customizable select is now in the spec"?

Reply:
Sorry for being slow - if I had to choose one of the PRs, I'd choose the base appearance rendering PR as the best indicator for the spec of customizable select: https://github.com/whatwg/html/pull/10629

Reply:
I've added them all in the meantime 😆
Thank you.